specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/privateLinkService.json

```diff
@@ -807,6 +807,11 @@
           },
           "description": "An array of private link service IP configurations."
         },
+        "destinationIPAddress": {
+          "readOnly": false,
+          "type": "string",
+          "description": "The destination IP Address for the PLS UDR Last Mile feature."
```

Comment:
Maybe this can be the description 

"The Destination IP Address for the private link service."

Reply:
Fixed